src/Provider/Settings.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' Settings plugin for Sylius.
+ *
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusSettingsPlugin\Provider;
+
+use MonsieurBiz\SyliusSettingsPlugin\Exception\SettingsException;
+use MonsieurBiz\SyliusSettingsPlugin\Settings\RegistryInterface;
+use Sylius\Component\Channel\Context\ChannelContextInterface;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+
+class Settings
```

Comment:
You don't have interface for your Provider ?

Reply:
Yep sorry, too early in the morning ☕